controllers/storagecluster/storageclient.go

```diff
@@ -0,0 +1,71 @@
+package storagecluster
+
+import (
+	"fmt"
+
+	ocsclientv1a1 "github.com/red-hat-storage/ocs-client-operator/api/v1alpha1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	cutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	storageClientName = "ocs-storagecluster-client"
```

Comment:
How do you know that storagecluster is going to be named `ocs-storagecluster`? 
I believe this is decided on the UI layer. You need to read the name from the instance and add the client suffix

Reply:
done.